pkgs/shells/rush/default.nix

```diff
@@ -8,7 +8,12 @@ stdenv.mkDerivation rec {
     sha256 = "1vxdb81ify4xcyygh86250pi50krb16dkj42i5ii4ns3araiwckz";
   };
 
-  patches = [ ./fix-format-security-error.patch ];
+  patches = [ ./fix-format-security-error.patch
+    (fetchpatch {
+      url = "https://sources.debian.org/data/main/r/rush/1.8+dfsg-1.1/debian/patches/intprops.patch";
```

Comment:
Unfortunately this is not a stable URL. When Debian releases new versions this version will go away. As the packaging it not in version control either, we should copy the patch into nixpkgs.